iceoryx_utils/test/moduletests/test_atomic_relocatable_ptr.cpp

```diff
@@ -156,19 +156,20 @@ TEST_F(AtomicRelocatablePointer_test, dereferenceOperator)
 // copied location
 TEST_F(AtomicRelocatablePointer_test, memoryRelocation)
 {
-    Memory<1024> memory;
+    constexpr uint64_t ALIGNMENT_OF_PTR {alignof(Ptr<byte_t>)};
+    Memory<1024, ALIGNMENT_OF_PTR> memory;
     memory[1000] = 37;
     // EXPECT_EQ(memory[1000], 37);
 
-    Ptr<byte_t>* rp = new (memory.ptr(10)) Ptr<byte_t>(memory.ptr(1000));
+    Ptr<byte_t>* rp = new (memory.ptr(ALIGNMENT_OF_PTR)) Ptr<byte_t>(memory.ptr(1000));
 
     // we have constructed a relocatable pointer at adress 10 in memory which points
```

Comment:
the comment is now out of sync with the code, we construct rp now at ALIGNMENT_OF_PTR which is the way to go since it will be aligned correctly this way

Reply:
upsi ... corrected :)